armadillo/http/test-storage.http

```diff
@@ -0,0 +1,51 @@
+### Authenticate
+GET {{host}}/storage/projects
+Authorization: Basic admin admin
+
+### List projects
+GET {{host}}/storage/projects
+
+### Create project
+POST {{host}}/storage/projects
+Content-Type: application/json
+
+{ "name" : "my-project" }
+
+### Project exists?
+HEAD {{host}}/storage/projects/my-project
+
+### Delete project
+DELETE {{host}}/storage/projects/my-project
+
+### List objects
+GET {{host}}/storage/projects/lifecycle/objects
+
+### Upload object
+POST {{host}}/storage/projects/lifecycle/objects
+Content-Type: multipart/form-data; boundary=boundary
+
+--boundary
+Content-Disposition: form-data; name="file"; filename="core/dummy.parquet"
+
+< ./dummy.parquet
+
+### Copy object
+POST {{host}}/storage/projects/lifecycle/objects/core%2Fnonrep.parquet/copy
```

Comment:
huh, do we really need copy and move? is that not a bit over engineering?

Reply:
They were present in the proposal I sent you. The client does provide copy and move, and my thinking was: better to let the backend manage this than spreading the operation over multiple requests and making the client download and upload the file again.